### ISSUE TITLE
about noisy results from own dataset

### ISSUE THREAD

**Issue by nuoma** [apchenstu/TensoRF#37]
Hi, I have created my own dataset but the results looks like this, do you have any idea why so?

![image](https://user-images.githubusercontent.com/9259412/192989275-0148b4ad-53ee-4f39-8ff7-faf81deed920.png)

The way I prepared my data is to capture ~200 images 360 degrees around a small object, then run colmap2nerf and split into train and test set. I probably need to segment out the object itself like tanks and temple dataset but didn't do so due to time constraint, but I used a clean white background.  Training process takes around 1hour, and reported train psnr = 24 test psnr = 12 mse = 0.003

thanks!

**Comment by basit-7** [comment]
Try reducing the aabb scale, it helped in my case.

**Comment by zjwzcx** [comment]
@nuoma Hi! Have you solved the problem? 
I successfully trained TensoRF on my own dataset including a object with regular shape and simple texture, nearly without any modification of hyper-parameter.
However, I met the same situation with you on my own dataset including a complicate object. I tried to modify: 1) n_iters, 2) near_far; 3) aabb; 4) N_voxel (grid_size), but no tricks can significantly improve the test PSNR. Could you please share some insight about setting hyper-parameter?

**Comment by RahulSajnani** [comment]
I am having the same issue. Please let me know if anyone has solved this issue

**Comment by heyjianjing** [comment]
from my experience, if your dataset has a scene that is large but still bounded, try to increase aabb and far, usually the algorithm gets me reasonably good results (resolution is not ideal since grids are more spread out), it would be slower. But giving it time, I still manage to get results close to instant-npg or nerfacto from nerfstudio.

however, if the scene is unbounded, or very large as compared to the size of the object in the middle you want to render, i have not managed to get good results. this is also the limitation mentioned in the paper.

for real scene,  you won't get ridiculously high psnr as in blender dataset, 29ish is the highest I can get so far

**Comment by yashbhalgat** [comment]
Hi @nuoma, I am facing similar problems with the ScanNet dataset. Have you found a solution?

**Comment by HannahHaensen** [comment]
@yashbhalgat have you solved the problem?

**Comment by emjay73** [comment]
same here

**Comment by yashbhalgat** [comment]
https://github.com/apchenstu/TensoRF/issues/46#issuecomment-1589255689

**Comment by BIT-DYN** [comment]
> from my experience, if your dataset has a scene that is large but still bounded, try to increase aabb and far, usually the algorithm gets me reasonably good results (resolution is not ideal since grids are more spread out), it would be slower. But giving it time, I still manage to get results close to instant-ngp or nerfacto from nerfstudio.
> 
> however, if the scene is unbounded, or very large as compared to the size of the object in the middle you want to render, i have not managed to get good results. this is also the limitation mentioned in the paper.
> 
> for real scene, you won't get ridiculously high psnr as in blender dataset

You saved me !!
